configure.ac

```diff
@@ -1640,33 +1667,34 @@ esac
 
 echo
 echo "Options used to compile and link:"
-echo "  with wallet   = $enable_wallet"
-echo "  with gui / qt = $bitcoin_enable_qt"
+echo "  with wallet     = $enable_wallet"
+echo "  with gui / qt   = $bitcoin_enable_qt"
 if test x$bitcoin_enable_qt != xno; then
-    echo "    with qr     = $use_qr"
+    echo "    with qr       = $use_qr"
 fi
-echo "  with zmq      = $use_zmq"
-echo "  with test     = $use_tests"
+echo "  external signer = $use_external_signer"
```

Comment:
In commit "configure: add --enable-external-signer" (ec6af5d1c993d584861be4ce6a0f5188f1af73af)

This is pretty misleading to print before the feature actually implemented. Would suggest commenting this out by default to avoid confusion.

Reply:
I commented this line out, but leaving `--enable-external-signer` in place (and keeping the indentation change).